### DISCUSSION TITLE
[Issue] Regarding Profile Page

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34375] [Projects and Issues]
In Windows 10, when I scroll down the profile page, the section of profile photo is overlapping the header part 

![image](https://user-images.githubusercontent.com/106828662/192606140-a0978236-7d28-4198-b95a-4aa3a15808b6.png)

and which is looking too much annoying, like readme is behind the header and profile part is overlapping, **In my opinion it should be behind that header** 

after all decision is yours GitHub, do you think it's looking good, have a clearer screenshot

![image](https://user-images.githubusercontent.com/106828662/192606311-48431958-983c-46fa-bf19-3028ac7a31d5.png)

**Comment by WHAHA-HA** [comment]
That's an excellent opinion. This is also something I think should be considered by the Github development team.

**Reply by ghost** [reply]
Still, nobody cares 😴

**Comment by jfullstackdev** [comment]
it's very common in websites where, like in Excel, it's Freeze Pane, so you will not lose sight of the Column Headers

same goes for Twitter as you can see here

![image](https://user-images.githubusercontent.com/47092464/193404460-40933b29-0911-4696-a464-fcb1e7757b7c.png)

![image](https://user-images.githubusercontent.com/47092464/193404490-1fad01dd-8697-47aa-b33a-f64f78dc9163.png)
